packages/babel-preset-react-app/index.js

```diff
@@ -59,54 +70,63 @@ if (env === 'development' || env === 'test') {
     // Adds component stack to warning messages
     require.resolve('babel-plugin-transform-react-jsx-source'),
     // Adds __self attribute to JSX which React will use for some warnings
-    require.resolve('babel-plugin-transform-react-jsx-self')
+    require.resolve('babel-plugin-transform-react-jsx-self'),
   ]);
 }
 
 if (env === 'test') {
   module.exports = {
     presets: [
       // ES features necessary for user's Node version
-      [require('babel-preset-env').default, {
-        targets: {
-          node: 'current',
+      [
+        require('babel-preset-env').default,
```

Comment:
Unrelated to this PR, but is there any reason this isn't `require.resolve` sub the default?